cmd/analyze.go

```diff
@@ -50,6 +51,18 @@ var (
 		Required: false,
 		Value:    false,
 	}
+
+	BaselineReport = &cli.StringFlag{
+		Name:     "baseline-report",
+		Usage:    "Path to the baseline report",
+		Required: false,
+	}
+
+	CompareReportFlag = &cli.BoolFlag{
```

Comment:
Let's remove this flag. If someone provide the `baseline-report` then it automatically means that they want to compare.